src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,169 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private int capacity;
+    private int threshold;
+    private int size;
+    private double loadFactor;
+    private Node<K, V>[] table;
+    private final int defaultInitialCapacity = 16;
+    private final double defaultLoadFactor = 0.75f;
+
+    public MyHashMap() {
+        this.capacity = defaultInitialCapacity;
+        this.threshold = (int) (capacity * defaultLoadFactor);
+        this.loadFactor = defaultLoadFactor;
+        this.table = new Node[defaultInitialCapacity];
```

Comment:
make sure to get rid of unnecessary this

Reply:
OK